docs/console-api/openapi/segments.js

```diff
@@ -723,6 +745,155 @@ module.exports = {
+                    }
+                  }
+                },
+                "examples": {
+                  "example-1": {
+                    "value": "{\n    \"message\": \"Rules successfully updated\"\n}"
+                  }
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "status": {
+                      "type": "integer"
+                    },
+                    "message": {
+                      "type": "string"
+                    }
+                  },
+                  "x-examples": {
+                    "example-1": {
+                      "status": 400,
+                      "message": "Segment must be of type 'id_list' to be modified"
+                    }
+                  }
+                },
+                "examples": {
+                  "example-1": {
+                    "value": {
+                      "status": 400,
+                      "message": "Segment must be of type 'rule_based' to be modified on this endpoint"
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "../models/error_401.json"
+                }
+              }
+            }
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          }
+        },
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "name": {
+                      "type": "string"
+                    },
+                    "passPercentage": {
+                      "type": "integer"
+                    },
+                    "conditions": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "../models/condition.json"
+                      }
+                    }
+                  }
+                }
+              },
+              "examples": {
+                "example-1": {
+                  "value": [
+                    {
+                      "name": "App version check",
+                      "passPercentage": 10,
+                      "conditions": [
+                        {
+                          "type": "app_version",
+                          "operator": "version_gt",
+                          "targetValue": [
+                            "1.1.1"
+                          ]
+                        }
```

Comment:
where are we documenting all of the conditions that can be used across gates/segments/dyanmic configs/etc?

Reply:
There is no current documentation. It is my highest priority as of now